src/integrations/slack/utils.ts

```diff
@@ -16,12 +21,21 @@ export const isBotMessage = function (event: any): boolean {
     );
 };
 
-// Limit team name to 3 words
-const TEAM_NAME_WORDS_LIMIT = 3;
+// Limit team/department/division to 3 words
+const WORDS_LIMIT = 3;
 
-export const getTeamNameFromProfile = function (
-    profile: any
-): string {
+export const getTeamNameFromProfile = (profile: any): string => {
     const teamName = USER_PROFILE_FIELD_ID_TEAM && profile.fields && profile.fields[USER_PROFILE_FIELD_ID_TEAM]?.value ? profile.fields[USER_PROFILE_FIELD_ID_TEAM].value : "";
-    return teamName ? teamName.split(" ").splice(0,TEAM_NAME_WORDS_LIMIT).join(" ") : "";
-}
\ No newline at end of file
+    return teamName ? teamName.split(" ").splice(0,WORDS_LIMIT).join(" ") : "Unknown";
+}
+
+export const getDepartmentFromProfile = (profile: any): string => {
+    const departmentName = USER_PROFILE_FIELD_ID_DEPARTMENT && profile.fields && profile.fields[USER_PROFILE_FIELD_ID_DEPARTMENT]?.value ? profile.fields[USER_PROFILE_FIELD_ID_DEPARTMENT].value : "";
```

Comment:
This and similar instructions above and below can be simplified with optional chaining to 
```js
const departmentName = profile?.fields?.[USER_PROFILE_FIELD_ID_DEPARTMENT]?.value ? profile.fields[USER_PROFILE_FIELD_ID_DEPARTMENT].value : "";
```
If `?.[USER_PROFILE_FIELD_ID_DEPARTMENT]` will not exist I guess it will be equivalent to `USER_PROFILE_FIELD_ID_DEPARTMENT` not existing?

Reply:
Great feedback, thanks!